src/command_modules/azure-cli-policyinsights/azure/cli/command_modules/policyinsights/_params.py

```diff
@@ -0,0 +1,97 @@
+            c.argument(
+                'resource_group_name',
+                options_list=['--resource-group-name', '-g'],
+                arg_type=resource_group_name_type,
+                completer=get_resource_group_completion_list,
+                help='Resource group name.')
+            c.argument(
+                'resource',
+                options_list=['--resource', '-r'],
+                validator=validate_resource,
+                help='Resource id or resource name.')
+            c.argument(
+                'resource_provider_namespace',
+                options_list=['--resource-provider-namespace', '-n'],
+                completer=get_providers_completion_list,
+                help='Resource provider namespace.')
+            c.argument(
+                'resource_type',
+                options_list=['--resource-type', '-t'],
+                completer=get_resource_types_completion_list,
+                help='Resource type.')
+            c.argument(
+                'policy_set_definition_name',
+                options_list=['--policy-set-definition-name', '-p'],
+                completer=get_policy_set_completion_list,
+                help='Policy set definition name.')
+            c.argument(
+                'policy_definition_name',
+                options_list=['--policy-definition-name', '-d'],
+                completer=get_policy_completion_list,
+                help='Policy definition name.')
+            c.argument(
+                'policy_assignment_name',
+                options_list=['--policy-assignment-name', '-a'],
+                completer=get_policy_assignment_completion_list,
+                help='Policy assignment name.')
+            c.argument(
+                'from_value',
+                options_list=['--from'],
+                help='ISO 8601 formatted timestamp specifying the start time of the interval to query.')
+            c.argument(
+                'to_value',
+                options_list=['--to'],
+                help='ISO 8601 formatted timestamp specifying the end time of the interval to query.')
+            c.argument(
+                'top_value',
+                options_list=['--top'],
+                type=int,
+                help='Maximum number of records to return.')
+            c.argument(
+                'order_by_clause',
+                options_list=['--order-by'],
+                help='Ordering expression using OData notation.')
+            c.argument(
+                'select_clause',
+                options_list=['--select'],
+                help='Select expression using OData notation.')
+            c.argument(
+                'filter_clause',
+                options_list=['--filter'],
+                help='Filter expression using OData notation.')
+            c.argument(
+                'apply_clause',
+                options_list=['--apply'],
+                help='Apply expression for aggregations using OData notation.')
```

Comment:
Use the `arg_group` kwarg to group similar arguments. Ex. "Resource Id", "Query", "Policy", etc.

Reply:
Will do